conda/environments/cuml_dev_cuda10.1.yml

```diff
@@ -26,3 +26,8 @@ dependencies:
 - libcumlprims=0.12*
 - statsmodels
 - protobuf >=3.4.1,<4.0.0
+- pip
+- pip:
+    - git+https://github.com/dask/dask.git
+    - git+https://github.com/dask/distributed.git
+    - git+https://github.com/dask/dask-ml.git
```

Comment:
I know this dep has been kicking around for a while, but do we actually use dask-ml anywhere? I couldn't find it.

Reply:
we might not be using it now that I look for it, I didn't check for this PR, do you think we should remove it?